Source/Core/DolphinQt2/Config/Mapping/GCPadEmu.cpp

```diff
@@ -43,10 +43,14 @@ void GCPadEmu::CreateMainLayout()
   // TODO: Get rid of this garbage once wx is removed
   // Remove "Iterative Input"
   auto* options_group = Pad::GetGroup(GetPort(), PadGroup::Options);
-  auto& bools = options_group->boolean_settings;
-  bools.erase(bools.end() - 1);
 
-  hbox_layout->addWidget(CreateGroupBox(tr("Options"), options_group));
+  if (options_group->boolean_settings.size() > 1 || options_group->boolean_settings.size() != 0)
```

Comment:
Wouldn't `!options_group->boolean_settings.empty()` be sufficient here instead of both of these checks?